client/src/reader/components/annotation/Share/Citation.js

```diff
@@ -59,7 +61,9 @@ export default class AnnotationShareEditor extends PureComponent {
 
     return (
       <fieldset className="citation__radios">
-        <legend className="citation__legend">Citation style:</legend>
+        <legend className="citation__legend">
+          {this.props.t("reader.citation_style") + ":"}
```

Comment:
Consider adding punctuation like this to the translation. I'm not sure all languages use ":" in the same way.

Reply:
Wanted to add, I haven't been consistent with this either, so it's on my list of things to go back and check for if we have time.